addons/sale_crm/views/sale_order_views.xml

```diff
@@ -31,10 +31,14 @@
             <field name="user_id" position="after">
                 <field name="tag_ids" widget="many2many_tags"/>
             </field>
+            <xpath expr="//group[@name='utm_link']" position="replace">
+                <group string="Marketing" name="utm_link" groups="base.group_no_one">
+                    <field name="campaign_id" groups="base.group_no_one" />
```

Comment:
No need to specify the group_no_one on the child nodes

Reply:
I removed group_no_one on the child nodes. Thanks